service/src/test/java/tds/assessment/repositories/StrandQueryRepositoryImplIntegrationTests.java

```diff
@@ -0,0 +1,137 @@
+                "0,1,4,4,1,1,NULL,NULL,2,3,NULL,'fixedform',NULL,5,1,20,1,5,'(SBAC_PT)SBAC-Mathematics-8-Spring-2013-2015',1,0,1,8185,8185,5,0,'SBAC_PT',NULL,'ABILITY',NULL,1,NULL,1,1,NULL,NULL,0,0,0,0," +
+                "0,'bp1',NULL,NULL,'summative');";
+        // Segment2
+        final String tblSetOfAdminSubjectsInsertSQL2b = "INSERT INTO itembank.tblsetofadminsubjects VALUES ('(SBAC_PT)SBAC-SEG2-MATH-8-Spring-2013-2015','SBAC_PT', 'SBAC_PT-ELA','SBAC-SEG2-MATH-8'," +
+                "0,1,4,4,1,1,NULL,NULL,1,4,NULL,'fixedform',NULL,5,1,20,1,5,'(SBAC_PT)SBAC-Mathematics-8-Spring-2013-2015',2,0,1,8185,8185,5,0,'SBAC_PT',NULL,'ABILITY',NULL,1,NULL,1,1,NULL,NULL,0,0,0,0," +
+                "0,'bp1',NULL,NULL,'summative');";
+
+        final String insertStrandSQL =
+                "INSERT INTO itembank.tbladminstrand\n" +
+                        "   (_fk_adminsubject, _key, _fk_strand, minitems, maxitems, adaptivecut)\n" +
+                        "VALUES \n" +
+                        "   ('(SBAC_PT)IRP-Perf-ELA-11-Summer-2015-2016', 'Strand1-key', 'Strand1', 2, 7, -39.234),\n" +
+                        "   ('(SBAC_PT)IRP-Perf-ELA-11-Summer-2015-2016', 'Strand2-key', 'Strand2', 0, 4, -37.432),\n" +
+                        "   ('(SBAC_PT)SBAC-Mathematics-8-Spring-2013-2015', 'Strand3-key', 'Strand3', 0, 2, null),\n" +
+                        "   ('(SBAC_PT)SBAC-SEG1-MATH-8-Spring-2013-2015', 'Strand4-key', 'Strand4', 1, 3, null),\n" +
+                        "   ('(SBAC_PT)SBAC-SEG2-MATH-8-Spring-2013-2015', 'Strand5-key', 'Strand5', 0, 4, null)";
+
+        jdbcTemplate.update(tblSetOfAdminSubjectsInsertSQL1);
+        jdbcTemplate.update(tblSetOfAdminSubjectsInsertSQL2);
+        jdbcTemplate.update(tblSetOfAdminSubjectsInsertSQL2a);
+        jdbcTemplate.update(tblSetOfAdminSubjectsInsertSQL2b);
+        jdbcTemplate.update(insertStrandSQL);
+    }
+
+    @Test
+    public void shouldFindBothStrandsForNonSegmented() {
+        final String assessmentKey = "(SBAC_PT)IRP-Perf-ELA-11-Summer-2015-2016";
+        Set<Strand> strands = repository.findStrands(assessmentKey);
+        assertThat(strands).hasSize(2);
+
+        Strand strand1 = null;
+        Strand strand2 = null;
+
+        for (Strand strand : strands) {
+            if ("Strand1".equals(strand.getName())) {
+                strand1 = strand;
+            } else if ("Strand2".equals(strand.getName())) {
+                strand2 = strand;
+            }
+        }
+
+        assertThat(strand1.getKey()).isEqualTo("Strand1-key");
+        assertThat(strand1.getMinItems()).isEqualTo(2);
+        assertThat(strand1.getMaxItems()).isEqualTo(7);
+        assertThat(strand1.getSegmentKey()).isEqualTo(assessmentKey);
+        assertThat(strand1.getAdaptiveCut()).isEqualTo(-39.234F);
+        assertThat(strand2.getKey()).isEqualTo("Strand2-key");
```

Comment:
Can you add a line break between the assertions on strand 1 and strand 2?  Makes it easier to read.